__tests__/fixtures/test-v11-prototype/app/assets/javascripts/application.js

```diff
@@ -1,10 +1,22 @@
 /* global $ */
+/* global GOVUK */
 
 // Warn about using the kit in production
 if (window.console && window.console.info) {
   window.console.info('GOV.UK Prototype Kit - do not use for production')
 }
 
 $(document).ready(function () {
-  window.GOVUKFrontend.initAll()
+  // Use GOV.UK shim-links-with-button-role.js to trigger a link styled to look like a button,
```

Comment:
You've removed `initAll()` from this test, I think it's important that it's still in.